test/e2e/e2e.go

```diff
@@ -188,15 +190,21 @@ func runPackageTest(pkgName, version string) error {
 	if err != nil {
 		return err
 	}
+
 	return nil
 }
 
 func runGinkgo() error {
 	var stdout bytes.Buffer
 	var stderr bytes.Buffer
+	mwriter := io.MultiWriter(os.Stdout)
+
 	cmd := exec.Command("ginkgo")
 	cmd.Stdout = &stdout
 	cmd.Stderr = &stderr
+
+	cmd.Stderr = mwriter
+	cmd.Stdout = mwriter
```

Comment:
@aman556 I didn't get what we are trying to do as part of this change with `io.MultiWriter`